include/unifex/sender_concepts.hpp

```diff
@@ -103,7 +105,9 @@ namespace detail {
 
   template <typename S>
   struct _typed_sender_traits {
-    template <template <typename...> class Tuple, template <typename...> class Variant>
+    template <
+        template <typename...> class Variant,
+        template <typename...> class Tuple>
     using value_types = typename S::template value_types<Tuple, Variant>;
```

Comment:
Still need to update this type alias to `...template value_types<Variant, Tuple>;`

Reply:
Good catch.